components/layout/formatting_contexts.rs

```diff
@@ -269,6 +275,35 @@ impl IndependentFormattingContext {
         preferred_aspect_ratio: Option<AspectRatio>,
         lazy_block_size: &LazySize,
     ) -> CacheableLayoutResult {
+        if self
```

Comment:
Replace this block to new match enum`IndependentFormattingContextContents::ReplacedWithWidget`?